sourcegraph/context.go

```diff
@@ -99,6 +100,8 @@ var NewClientFromContext = func(ctx context.Context) *Client {
 			creds = credentials.NewTLS(&tls.Config{InsecureSkipVerify: true})
 		}
 		opts = append(opts, grpc.WithTransportCredentials(creds))
+	} else {
+		opts = append(opts, grpc.WithInsecure())
```

Comment:
The grpc-go API now expects clients to explicitly specify whether TLS is configured on the underlying transport. The `WithInsecure` DialOption is for supporting instances that run without TLS.